examples/basic/main.tf

```diff
@@ -10,6 +10,12 @@ module "roks_landing_zone" {
   resource_tags    = var.resource_tags
 }
 
+resource "time_sleep" "wait_5_minutes" {
+  depends_on = [module.roks_landing_zone]
+
+  destroy_duration = "300s"
+}
```

Comment:
@jor2 I see the sleep as a temp workaround - we still need to figure out what resource(s) are taking time to clean up. I suspect the cluster is creating something (maybe a load balancer).
When we figure out what is taking time, we should include the details in the code comment (or link to a git issue with the details)